run-tests.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# Try to install composer dev dependencies
+composer install --no-interaction --no-scripts --no-plugins
+
+# If that failed, exit.
+rc=$?; if [[ $rc != 0 ]]; then exit $rc; fi
+
+# Run the feature tests
+vendor/bin/behat --stop-on-failure
```

Comment:
Oh, does the return value / output of the last command in a bash script end up being the return value / output when the bash script itself is run? If so, is there anything else runny would gain us that is worth having here (like logging to syslog)?